migration/migrator/migrations/course/20190710182514_add_view_date_for_teams.py

```diff
@@ -0,0 +1,5 @@
+"""Migration for a given Submitty course database."""
+
+def up(config, database, semester, course):
+    if not database.table_has_column('teams', 'last_viewed_time'):
+        database.execute('ALTER TABLE teams ADD COLUMN last_viewed_time timestamp with time zone')
```

Comment:
Please add a down function with an empty body (just 'pass')
See other migrations for examples.

Reply:
Added in last commit.

Reply:
Added in last commit.